packages/core/src/options.ts

```diff
@@ -107,6 +107,11 @@ export type CoreOptionsType = Required<
   Pick<Options, 'staticHelpers' | 'staticModifiers' | 'staticComponents' | 'staticInvokables'>;
 
 export function optionsWithDefaults(options?: Options): CoreOptionsType {
+  if ((options as any)?.skipBabel) {
```

Comment:
Just theoretically possible someone make skipBabel: false and this won’t detect the case so maybe could be
```suggestion
  if (‘skipBabel’ in (options as any) || {}) {
```

Reply:
oh that's nice thanks! Although this PR isn't ready because I was expecting CI to error because we have some places where this is being set 🙈

Reply:
oh turns out this doesn't work? 🤔 I just tried it with `staticEmberSource` and it's always reporting true:

```
  if ('staticEmberSource' in (options as any) || {}) {
    console.log({ options });
  }
```

That logs:

```
{
  options: {
    staticAddonTrees: true,
    staticAddonTestSupportTrees: true,
    allowUnsafeDynamicComponents: false,
    staticInvokables: true,
    amdCompatibility: { es: [] }
  }
}
```

Reply:
I've tested the new version and I'm happy that it's right now 👍